third_party/move/tools/move-formatter/tests/formatter/fun/input5.fmt.move

```diff
@@ -0,0 +1,55 @@
+module TestFunFormat {
+    struct S has store {}
+
+    struct R has store {}
+
+    struct T has store {}
+
+    struct G<T> has store {}
+
+    fun f1()
+        acquires S {}
+
+    fun f2()
+        reads S {}
+
+    fun f3()
+        writes S {}
+
+    fun f4()
+        acquires S(*) {}
+
+    fun f_multiple()
+        acquires R
+        reads R
+        writes T, S
+        reads G<u64> {}
+
+    fun f5()
+        acquires 0x42::*::* {}
+
+    fun f6()
+        acquires 0x42::m::* {}
+
+    fun f7()
+        acquires *(*) {}
+
+    fun f8()
+        acquires *(0x42) {}
+
+    fun f9(a: address)
+        acquires *(a) {}
+
+    fun f10(x: u64)
+        acquires *(make_up_address(x)) {}
+
+    fun make_up_address(x: u64): address {
+        @0x42
+    }
+
+    fun f11()
+        !reads *(0x42), *(0x43) {}
+
+    fun f12()
+        pure {}
+}
```

Comment:
I don't think we should remove the last empty line?

Reply:
I have tried rustfmt, and it automatically adds an extra line of whitespace. Can it be like rustfmt, which adds an extra line of whitespace if the last line of the source file is not a blank line, and if there are multiple blank lines at the end of the source file, it retains one line of whitespace?

Reply:
Yeah, I think we can follow what rustfmt does here.

Reply:
fixed